shell/platform/tizen/channels/text_input_channel.cc

```diff
@@ -57,6 +57,45 @@ static bool IsASCIIPrintableKey(char c) {
   return false;
 }
 
+static bool TextInputTypeToEcoreIMFInputPanelLayout(
+    std::string text_input_type,
+    Ecore_IMF_Input_Panel_Layout& panel_layout) {
+  FT_LOGI("text_input_type[%s]", text_input_type.c_str());
```

Comment:
Please use FT_LOGD for debug log messages until [the clean up](https://github.com/flutter-tizen/engine/pull/75#discussion_r628036964) is done.

Reply:
Sorry, it's my mistake. I didn't intend, I'll remove this line